app/views/shared/_supporting_information.html.erb

```diff
@@ -65,16 +65,22 @@
       </h2>
     </div>
     <div id="accordion-default-content-3" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-3">
-      <% if @agent %>
+      <% if @planning_application.agent_first_name %>
```

Comment:
Might be a bit more accurate if we have an `agent?` and `applicant?` method on the PlanningApplication model that returns true if first_name, last_name, phone *or* email is present?

Reply:
Good idea 👍

Reply:
Fixed on last commit.